source/ghost-text.js

```diff
@@ -196,9 +203,14 @@ class GhostTextField {
 		this.field.removeEventListener('input', this.send);
 		this.field.dataset.gtField = '';
 
-		chrome.runtime.sendMessage({
-			code: 'focus-tab',
-		});
+		(async () => {
```

Comment:
Same here. You can try turning `deactivate` async instead

Reply:
done